src/App.tsx

```diff
@@ -1,61 +1,114 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
```

Comment:
Ensure that the `usersFromServer` and `todosFromServer` imports are correctly defined and exported from their respective files. Verify their existence and correct data structure.